### PR TITLE
Unify varchar limits

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | PR to update DB structure after https://github.com/PrestaShop/PrestaShop/pull/35882.
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      |

### CODE REVIEW

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @Hlavtox 

Thank you for your PR, I tested it and it seems to works as you can see : 

image

image

image

image

image

image

Screen taken from 8.0.3 upgrade to 9.0.0 and from a fresh 9.0.0

Tested from : 
1.7.8.9 to 8.1.7 
8.0.5 to 8.1.7 
8.1.5 to 8.1.7 
8.1.7 to 9.0.0
8.1.5 to 9.0.0
8.0.5 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
